scripts/dogCard.js

```diff
@@ -0,0 +1,9 @@
+
+
+$('.dogcard').hover(function() {
+    $( this ).toggleClass('dogcard-focus');
```

Comment:
would try using addClass removeClass to see if this stops the tiles from glitching when you hover over them https://api.jquery.com/hover/